teleband/courses/api/views.py

```diff
@@ -236,7 +237,7 @@ def roster(self, request, **kwargs):
             course_enrollments, many=True, context={"request": request}
         )
         return Response(status=status.HTTP_200_OK, data=serializer.data)
-    
+
```

Comment:
@Adamv27 perhaps you can add something to the backlog about having us create a linter config file? this PR is bigger than necessary because of distracting whitespace changes...